web-client/src/views/CaseDetail/AddIrsPractitionerModal.tsx

```diff
@@ -34,6 +34,7 @@ export const AddIrsPractitionerModal = connect(
         className="counsel-modal"
         confirmLabel="Add to Case"
         confirmSequence={confirmSequence}
+        dataTestId="irs-practitioner-modal"
```

Comment:
I think this should be `data-test-id`

Reply:
it's a prop on Modal which is why it's like that.